pymatgen/io/lammps/output.py

```diff
@@ -2,22 +2,7 @@
 # Copyright (c) Pymatgen Development Team.
 # Distributed under the terms of the MIT License.
 
-from __future__ import division, print_function, unicode_literals, \
-    absolute_import
-
-import re
-from io import open
-import os
-
-import numpy as np
-
-from monty.json import MSONable
-
-from pymatgen.core.periodic_table import _pt_data
-from pymatgen.core.structure import Structure
-from pymatgen.core.lattice import Lattice
-from pymatgen.analysis.diffusion_analyzer import DiffusionAnalyzer
-from pymatgen.io.lammps.data import LammpsData, LammpsForceFieldData
+from __future__ import division, print_function, unicode_literals, absolute_import
```

Comment:
I will get to those at some point in time (e.g, the way I got to the GW input sets). 

The fact that there are some code that do not obey the rules does not mean that all code do not obey rules. :-)

P.S. Try maintaining a full code one day and you will see the difficulty.